srcpkgs/keybase-desktop/template

```diff
@@ -0,0 +1,53 @@
+# Template file for 'keybase-destkop'
+pkgname=keybase-desktop
+version=2.6.2
+revision=1
+wrksrc="client-${version}"
+hostmakedepends="git nodejs-lts yarn"
+depends="keybase kbfs"
+short_desc="Keybase desktop client"
+maintainer="Dominic Monroe <monroef4@googlemail.com>"
+license="BSD-3-Clause"
+homepage="https://keybase.io"
+distfiles="https://github.com/keybase/client/archive/v${version}.tar.gz"
+checksum=79e72947908b1e7983ab2395d06d66c8ac93bebd28850674929fdc41690d0e15
+# Electron/Keybase Electron limitation
+only_for_archs="x86_64  x86_64-musl i686 i686-musl"
```

Comment:
I'd rather a `broken=` clause instead, maybe as part of the case statement below?

Reply:
Since electron probably wants to be more cross platform, just is stupid about it. arm is not out of scope.

Reply:
I'm not sure if this changes your mind at all, but keybase only provide base builds for these platforms: https://keybase.pub/kbelectron/electron-download/v2.0.8/

Reply:
So did you try other arches?